src/lib.rs

```diff
@@ -68,6 +72,19 @@ impl CaptureIndex {
 
 const ALL_NODES_QUERY: &str = "(_) @node";
 
+fn join_with_or<TItem: fmt::Display>(list: &[TItem]) -> String {
+    let mut ret: String = Default::default();
+    for (index, item) in list.iter().enumerate() {
+        ret.push_str(&item.to_string());
+        if list.len() >= 2 && index < list.len() - 2 {
+            ret.push_str(", ");
+        } else if list.len() >= 2 && index == list.len() - 2 {
+            ret.push_str(" or ");
```

Comment:
I vote for an Oxford comma (?): "`, or " 😄

Reply:
Sure

I special-cased the `list.len() == 2` case because I assume you wouldn't say eg "Typescript, or Javascript" (instead in that case "Typescript or Javascript")?

Oh also if we want to get even more posh I slightly considered using "nor" instead of "or" here